docs/new_analyzer.md

```diff
@@ -0,0 +1,71 @@
+# How to add a new analyzer on the code review bot
```

Comment:
I'd split the two things in two files, one saying how to enable static analysis on your repo, and one saying how to add a new analyzer.

Reply:
I'm not sure it's worth ir: most people i talked too want to do the full thing

Reply:
Mmmh, aren't most people interested in adding new analyzers to mozilla-central?

Reply:
I see what you mean, i'll split it